.github/workflows/cherry-pick.yml

```diff
@@ -0,0 +1,112 @@
+name: Cherry pick
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - 7.x.x
```

Comment:
Does this need to be `main`?

Reply:
No, this workflow will be dormant in `main` and only used on LTS branches.